### PR TITLE
[coding] [geometry] Deprecated PointToInt.

### PR DESCRIPTION
**Pull request by mpimenov**
For some reason we have been using bitwise merge to store
a pair of 32-bit unsigned integer coordinates. Since the
width of the coordinates is fixed and storage in general
has nothing to do with the Z-order curve where bitwise
merge is appropriate, this commit marks this method of
storage as obsolete (and effectively deprecated).

The functions to convert between PointD and PointU are
still serviceable. Their usage is slightly refactored.

### CODE REVIEW

**Comment by Zverik** [comment]
1. Do we change the mwm format to stop using the z-merge of coordinates?
2. Did you test that removing these methods do not drastically increase size of mwm?
3. Are you sure storing two [small] uints separately takes less space that storing one, produced by z-merging these uints?

**Comment by mpimenov** [comment]
@Zverik 
1. No, this PR only refactors the existing code. The goal is to discourage future users from using it when a need to serialize points arises.
2. No methods are removed.
3. I'm sure even if it does not, the loss is negligible. I'm also sure it is two times faster cpu-wise.

File: coding/coding_tests/pointd_to_pointu_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}  // namespace
+
+UNIT_TEST(PointDToPointU_Epsilons)
+{
+  m2::PointD const arrPt[] = {{-180, -180}, {-180, 180}, {180, 180}, {180, -180}};
+  m2::PointD const arrD[] = {{1, 1}, {1, -1}, {-1, -1}, {-1, 1}};
+  size_t const count = ARRAY_SIZE(arrPt);
+
+  /*
+  double eps = 1.0;
```
[the remark this answers is not in the document]
Comment (mpimenov) [reply coding/coding_tests/pointd_to_pointu_tests.cpp]: Какой-то старый код, который отлаживал эпсилоны. Можно и убрать.

**Comment by mpimenov** [comment]
JTTP

**Comment by mpimenov** [comment]
PTAL

**Pull request merged**